doc/sphinx/source/index.rst

```diff
@@ -13,32 +13,87 @@
 .. limitations under the License.
 .. ---------------------------------------------------------------------------
 
-###############
-nGraph library
-###############
 
+.. This documentation is available online at 
+.. http://ngraph.nervanasys.com/docs/latest/
 
-Welcome to nGraph™, an open-source C++ compiler library for running and 
-training :abbr:`Deep Neural Network (DNN)` models. This project is 
-framework-neutral and can target a variety of modern devices or platforms. 
+
+########
+nGraph™ 
+########
+
+Welcome to the documentation site for nGraph™, an open-source C++ library and 
+command-line suite for the :abbr:`Deep Learning (DL)` ecosystem. Our goal with 
```

Comment:
I don't think we should say "command-line suite" at this point. The only real command-line tool we have is `nbench`, which is a benchmarking tool for serialized models.

Reply:
I was looking for a more succinct way to say "API / compiler / runtime" .  To me all of those things comprise the current version of our command-line suite.